app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -90,7 +93,7 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
     private var resourceURL: String = ""
     private var unlocked = false
 
-    @SuppressLint("SetJavaScriptEnabled")
+    @SuppressLint("SetJavaScriptEnabled", "RestrictedApi")
```

Comment:
Usually not a good sign if we have to disable the linter from warning about using a restricted api. Which one is a restricted api here ?

Reply:
It was for `FORCE_DARK_STRATEGY` I assume because its new?

Reply:
Would github actions fail if we submitted without `RestrictedApi`? It does run fine locally when I remove it.  I only added it so Android Studio wouldn't complain :)

Reply:
@dshokouhi I removed it locally and I didn't get the linter complaining.

Reply:
Yup it didn't complain, probably stale code on my end...my bad :)